src/actions/actions.ts

```diff
@@ -2190,11 +2188,11 @@ class MoveToMatchingBracket extends BaseMovement {
       let matchedPosition: Position | undefined = undefined;
 
       for (const { char, pos } of Position.IterateDocument(position, toFind.nextMatchIsForward)) {
-        if (char === charToMatch) {
+        if (char === charToMatch && charToMatch !== toFind.match) {
           stackHeight++;
         }
 
-        if (char === this.pairings[charToMatch].match) {
+        if (char === toFind.match) {
```

Comment:
Yes, I plan on making the class generic enough so that it can handle `a<`, `i)` etc (and may add them to this PR now that I have some better confidence I'm going down the right track).

Will come up with some better naming, and add `'`/`"` into `pairings`